src/syntax/ppx_sedlex.ml

```diff
@@ -311,16 +457,37 @@ let regexp_of_pattern env =
       | _ ->
           err p.ppat_loc @@ "the " ^ name
           ^ " operator requires two arguments, like " ^ name ^ "(a,b)"
-  and aux p =
+  and aux ?(allow_alias = false) p =
```

Comment:
Make allow_alias non optionnal. It will help to carefully review all recursive calls.